### PR TITLE
Added support for apt pakage manager

### LINKED ISSUE #18: Making Makefile on Ubuntu gives error
sudo: dnf: command not found
Makefile:5: recipe for target 'install-dependencies' failed
make: *** [install-dependencies] Error 1

Screenshot from 2020-03-03 04-27-09

### PR DESCRIPTION
**Pull request by ritwickraj78**
Fixes #18

### CODE REVIEW

**Comment by ritwickraj78** [comment]
Referencing the issue #18

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,14 @@
 CURDIR := ${CURDIR}
 IMAGE ?= usercont/packit-dashboard:stg
-
 install-dependencies:
-	sudo dnf -y install python3-flask npm
+	if [ -f "/etc/redhat-release" ];\
+	 then\
+		sudo dnf -y install python3-flask npm;\
```
Comment (lachmanfrantisek) [inline Makefile]: What about having the packages in a variable so it's prepared for easy updates.

Comment (ritwickraj78) [reply Makefile]: I'll do that ASAP. Thanks

Comment (lachmanfrantisek) [reply Makefile]: Thanks.

**Pull request merged**